ui-configuration.graphqls

```diff
@@ -21,7 +21,10 @@ enum TemplateType {
     TOPOLOGY_INSTANCE,
     TOPOLOGY_ENDPOINT,
     TOPOLOGY_SERVICE_RELATION,
-    TOPOLOGY_SERVICE_INSTANCE_RELATION
+    TOPOLOGY_SERVICE_INSTANCE_RELATION,
+    TOPOLOGY_ENDPOINT_DEPENDENCY,
+    TOPOLOGY_SERVICE_DEPENDENCY,
```

Comment:
What is the different between relationship and dependency?

Reply:
> What is the different between relationship and dependency?

Overlooked those `*_RELATION`, so they were not used and now can be used.